iocore/net/UnixNet.cc

```diff
@@ -69,7 +69,25 @@ class InactivityCop : public Continuation
         continue;
       }
 
+      // set a default inactivity timeout if one is not set
+      bool use_default_inactivity_timeout = false;
+      if (ne->next_inactivity_timeout_at == 0 && nh.config.default_inactivity_timeout > 0) {
+        Debug("inactivity_cop", "vc: %p inactivity timeout not set, setting a default of %d", ne,
+              nh.config.default_inactivity_timeout);
+        use_default_inactivity_timeout = true;
+        ne->set_inactivity_timeout(HRTIME_SECONDS(nh.config.default_inactivity_timeout));
+        NET_INCREMENT_DYN_STAT(default_inactivity_timeout_applied_stat);
+      } else {
+        Debug("inactivity_cop_verbose", "vc: %p now: %" PRId64 " timeout at: %" PRId64 " timeout in: %" PRId64, ne,
+              ink_hrtime_to_sec(now), ink_hrtime_to_sec(ne->next_inactivity_timeout_at),
+              ink_hrtime_to_sec(ne->inactivity_timeout_in));
+      }
+
       if (ne->next_inactivity_timeout_at && ne->next_inactivity_timeout_at < now) {
+        if (use_default_inactivity_timeout) {
+          // track the connections that timed out due to default inactivity
+          NET_INCREMENT_DYN_STAT(default_inactivity_timeout_count_stat);
```

Comment:
How is this going to trigger?  When you loop through inactivity cop and see that the next_inactivity_timeout_at == 0 you will set the default.  Then in the future when you loop through inactivity cop again and you hit the timeout use_default_inactivity_timeout should be false.

Reply:
Ah, that's true - I meant to reset the timeout back to 0 after checking if it expired but missed it. 

Makes sense to move this to a separate PR.